README.md

```diff
@@ -389,6 +390,52 @@ The secret can be created by executing the following command:
 oc create secret generic keycloak-group-sync --from-literal=username=<username> --from-literal=password=<password>
 ```
 
+### Okta
+
+Groups assigned to Okta applications can be synchronized into OpenShift. The following table describes the set of configuration options for the Okta provider:
+
+| Name | Description | Defaults | Required | 
+| ----- | ---------- | -------- | ----- |
+| `credentialsSecret` | Reference to a secret containing authentication details (See below) | | Yes |
+| `groups` | List of groups to filter against | | No |
+| `scope` | Scope for group synchronization. Options are `one` for one level or `sub` to include subgroups | `sub` | No |
+| `url` | URL Location for Okta | | Yes |
```

Comment:
Need to be explicit on the format that is required (scheme and trailing `/`)

Reply:
Updated this in the examples and added trailing slash requirement